scrapy/utils/conf.py

```diff
@@ -141,6 +142,13 @@ def check_valid_format(output_format):
                              " or as a file extension, from the supported list %s" %
                              (output_format, tuple(valid_output_formats)))
 
+    overwrite = False
+    if overwrite_output:
+        if output:
+            raise UsageError("Please use only one of --output and --overwrite-output")
```

Comment:
```suggestion
            raise UsageError("Please use only one of -o/--output and -O/--overwrite-output")
```
Or something similar. I wonder if that would be more informative, perhaps not everybody knows exactly what `-o/-O` means.